File: SERVER/foody-admin/src/admin/modules/user/food/Food.php
<?php

class Food
{
    private $id;
    private $foodCode;
    private $foodName;
    private $foodAddress;
    private $foodPrice;
    private $imageAddress;
    private $resCode;
    private $kindCode;
    private $create_at;
    private $update_at;

    // public function __construct($id, $foodCode, $foodName, $foodAddress, $foodPrice, $imageAddress)
    // {
    //     $this->ID = $id;
    //     $this->FoodCode = $foodCode;
    //     $this->FoodName = $foodName;
    //     $this->FoodAddress = $foodAddress;
    //     $this->FoodPrice = $foodPrice;
    //     $this->ImageAddress = $imageAddress;
    // }

    /**
     * Get the value of id.
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set the value of id.
     *
     * @return self
     */
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

    /**
     * Get the value of foodCode.
     */
    public function getFoodCode()
    {
        return $this->foodCode;
    }

    /**
     * Set the value of foodCode.
     *
     * @return self
     */
    public function setFoodCode($foodCode)
    {
        $this->foodCode = $foodCode;

        return $this;
    }

    /**
     * Get the value of foodName.
     */
    public function getFoodName()
    {
        return $this->foodName;
    }

    /**
     * Set the value of foodName.
     *
     * @return self
     */
    public function setFoodName($foodName)
    {
        $this->foodName = $foodName;

        return $this;
    }

    /**
     * Get the value of foodPrice.
     */
    public function getFoodPrice()
    {
        return $this->foodPrice;
    }

    /**
     * Set the value of foodPrice.
     *
     * @return self
     */
    public function setFoodPrice($foodPrice)
    {
        $this->foodPrice = $foodPrice;

        return $this;
    }

    /**
     * Get the value of imageAddress.
     */
    public function getImageAddress()
    {
        return $this->imageAddress;
    }

    /**
     * Set the value of imageAddress.
     *
     * @return self
     */
    public function setImageAddress($imageAddress)
    {
        $this->imageAddress = $imageAddress;

        return $this;
    }

    /**
     * Get the value of foodAddress.
     */
    public function getFoodAddress()
    {
        return $this->foodAddress;
    }

    /**
     * Set the value of foodAddress.
     *
     * @return self
     */
    public function setFoodAddress($foodAddress)
    {
        $this->foodAddress = $foodAddress;

        return $this;
    }

    /**
     * Get the value of resCode.
     */
    public function getResCode()
    {
        return $this->resCode;
    }

    /**
     * Set the value of resCode.
     *
     * @return self
     */
    public function setResCode($resCode)
    {
        $this->resCode = $resCode;

        return $this;
    }

    /**
     * Get the value of create_at.
     */
    public function getCreate_at()
    {
        return $this->create_at;
    }

    /**
     * Set the value of create_at.
     *
     * @return self
     */
    public function setCreate_at($create_at)
    {
        $this->create_at = $create_at;

        return $this;
    }

    /**
     * Get the value of update_at.
     */
    public function getUpdate_at()
    {
        return $this->update_at;
    }

    /**
     * Set the value of update_at.
     *
     * @return self
     */
    public function setUpdate_at($update_at)
    {
        $this->update_at = $update_at;

        return $this;
    }

    /**
     * Get the value of kindCode.
     */
    public function getKindCode()
    {
        return $this->kindCode;
    }

    /**
     * Set the value of kindCode.
     *
     * @return self
     */
    public function setKindCode($kindCode)
    {
        $this->kindCode = $kindCode;

        return $this;
    }
}

File: APP/Foody/app/src/main/java/com/gmail/khanhit100896/foody/food/Food.java
package com.gmail.khanhit100896.foody.food;

public class Food {
    private int id;
    private String food_code;
    private String food_name;
    private String food_address;
    private String food_price;
    private String img_address;
    private String res_code;
    private String kind_code;
    private int action_like;
    private int action_touch;
    private double distance;

    public Food(int id, String food_code, String food_name, String food_address, String food_price,
                String img_address, String res_code, String kind_code) {
        this.id = id;
        this.food_code = food_code;
        this.food_name = food_name;
        this.food_address = food_address;
        this.food_price = food_price;
        this.img_address = img_address;
        this.res_code = res_code;
        this.kind_code = kind_code;
    }

    public Food(int id, String food_code, String food_name, String food_address, String food_price,
                String img_address, String res_code, String kind_code, int action_like) {
        this.id = id;
        this.food_code = food_code;
        this.food_name = food_name;
        this.food_address = food_address;
        this.food_price = food_price;
        this.img_address = img_address;
        this.res_code = res_code;
        this.kind_code = kind_code;
        this.action_like = action_like;
    }

    public Food(int id, String food_code, String food_name, String food_address, String food_price,
                String img_address, String res_code, String kind_code, int action_like, double distance) {
        this.id = id;
        this.food_code = food_code;
        this.food_name = food_name;
        this.food_address = food_address;
        this.food_price = food_price;
        this.img_address = img_address;
        this.res_code = res_code;
        this.kind_code = kind_code;
        this.action_like = action_like;
        this.setDistance(distance);
    }

    public int getId() {
        return id;
    }

    public String getFood_code() {
        return food_code;
    }

    public String getFood_name() {
        return food_name;
    }

    public String getFood_address() {
        return food_address;
    }

    public String getFood_price() {
        return food_price;
    }

    public String getImg_address() {
        return img_address;
    }

    public String getRes_code() {
        return res_code;
    }

    public String getKind_code() {
        return kind_code;
    }

    public int getAction_like() {
        return action_like;
    }

    public void setAction_like(int action_like) {
        this.action_like = action_like;
    }

    public int getAction_touch() {
        return action_touch;
    }

    public void setAction_touch(int action_touch) {
        this.action_touch = action_touch;
    }

    public double getDistance() {
        return distance;
    }

    public void setDistance(double distance) {
        this.distance = distance;
    }
}
